x-pack/plugins/apm/public/components/alerting/rule_types/register_apm_rule_types.ts

```diff
@@ -33,7 +33,7 @@ export function registerApmRuleTypes(
       return {
         reason: fields[ALERT_REASON]!,
         link: getAlertUrlErrorCount(
-          String(fields[SERVICE_NAME][0]!),
+          String(fields[SERVICE_NAME]![0]),
```

Comment:
Why did this change?

Reply:
As mentioned elsewhere, it would be awesome to have proper types for alert and rule params.

Reply:
@dgieselaar, because the service name field is not required. This will be added as TODO now, and fixed in a separate PR when we move these APM specific fields to `initializeIndex`